AppCenterData/AppCenterData/Internal/Model/MSDataErrorInternal.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSDataError.h"
+
+@interface MSDataError ()
```

Comment:
You need the nullability declaration in this file. Please use `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`.